src/drt/src/pa/FlexPA_prep.cpp

```diff
@@ -1499,7 +1499,7 @@ int FlexPA::initPinAccess(T* pin, frInstTerm* inst_term)
     if (is_macro_cell_pin) {
       macro_cell_pin_no_ap_cnt_++;
     }
-  } else {
+  } else if (inst_term) {
```

Comment:
What if the inst_temr is nullptr? Should it be an error?

Reply:
The comment 
```
  // inst_term aps are written back here if not early stopped
  // IO term aps are are written back in prepPoint_pin_helper and always
  // early stopped
```
suggests that it is expected to be non-null here as the IO term (bterm) are handled separately